.github/workflows/ohos.yml

```diff
@@ -245,19 +237,38 @@ jobs:
           [[ $servo_pid =~ ^[0-9]+$ ]] || { echo "It looks like servo crashed!" ; exit 1; }
           # If the grep fails, then the trace output for the "page loaded" prompt is missing
           grep 'org\.servo\.servo-.* tracing_mark_write.*PageLoadEndedPrompt' test_output/servo.ftrace
-      - name: Getting runs file
-        uses: actions/download-artifact@v4
+
+  # speedometer ohos needs the command-line-tools and the repository for mach installed
+  # we require that this runs after test-harmonyos-aarch64 because then we know the binary is installed on the phone
+  bench-harmonyos-aarch64:
+    name: Benching HarmonyOS aarch64
+    continue-on-error: true
+    runs-on: hos-runner
+    needs: test-harmonyos-aarch64
+    if: github.repository == 'servo/servo'
+    steps:
+      - uses: actions/checkout@v4
+        if: github.event_name != 'pull_request_target'
         with:
-          # Name of the artifact to download.
-          # If unspecified, all artifacts for the run are downloaded.
-          name: runs.json
+          fetch-depth: 2
+      # This is necessary to checkout the pull request if this run was triggered via a
+      # `pull_request_target` event.
+      - uses: actions/checkout@v4
+        if: github.event_name == 'pull_request_target'
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 2
+      - name: Setup Python
+        uses: ./.github/actions/setup-python
       - name: "Run benchmark"
-        run: hitrace-bench -r runs.json
-      - name: Getting bencher
-        uses: bencherdev/bencher@main
+        run: hitrace-bench -r support/hitrace-bencher/runs.json
+      - name: Run speedometer
+        run: python3 ./mach test-speedometer-ohos --bmf-output speedometer.json
       - name: Getting model name
         run: |
           echo "MODEL_NAME=$(hdc bugreport | head -n 20 | grep MarketName | awk '{for (i=2; i<NF; i++) printf $i " "; print $NF}' -)" >> $GITHUB_ENV
-      - name: Uploading to bencher.dev
-        run: |
-          bencher run --adapter json --file bench.json --project '${{ env.BENCHER_PROJECT }}' --token '${{ secrets.BENCHER_API_TOKEN }}' --github-actions '${{ secrets.GITHUB_TOKEN }}' --testbed="$MODEL_NAME"
+      - name: Combining bencher files
+        run: jq -cs add speedometer.json bench.json >> combined-bencher.json
```

Comment:
Do `-cs` have long form variants of the flags? helps readability in ci a lot imho to prefer long form flags (unless its something super common).